File: prime/prime.jl
function sieve(n)
    primes = fill(true, n)
    primes[1] = false
    
    for i in 2:isqrt(n)
        if primes[i]
            for j in (i*i):i:n
                primes[j] = false
            end
        end
    end
    
    count = sum(primes)
    println("Found $count primes")
end

sieve(10_000_000)

File: prime/prime.py
def sieve(n):
    primes = [True] * (n + 1)
    primes[0] = primes[1] = False
    
    for i in range(2, int(n**0.5) + 1):
        if primes[i]:
            for j in range(i*i, n + 1, i):
                primes[j] = False
            
    count = sum(primes)
    print(f"Found {count} primes")

sieve(10000000)